tests/unit/test_views.py

```diff
@@ -290,6 +297,14 @@ def testVariantSetsSearch(self):
             response.data)
         self.assertEqual(len(responseData.variantSets), 1)
 
+    def testGetDataset(self):
+        response = self.sendDatasetsSearch()
+        responseData = protocol.SearchDatasetsResponse.fromJsonString(
+            response.data)
+        datasetId = responseData.datasets[0].id
+        response = self.sendGetDataset(datasetId)
+        self.assertEqual(200, response.status_code)
```

Comment:
We need to test the 'not found' condition here too, which should give us a 404 HTTP response. See the second half of testGetVariantSet for a model to follow.